src/pages/PingDetail.svelte

```diff
@@ -50,6 +51,14 @@
           /></a
         >
       {/if}
+      {#each ping.tags as tag}
```

Comment:
should we check if `ping.tags` exists first?

Reply:
I don't think such a check is needed: we should always have this property (although sometimes it will be an empty array)

Reply:
The loop will just not run if array is empty. You can see an example of this behaviour here: https://deploy-preview-967--glean-dictionary-dev.netlify.app/apps/mozilla_vpn/pings/main

Reply:
ah you're right!